src/Library.php

```diff
@@ -114,76 +110,4 @@ protected static function register_atomic_blocks() {
 			$instance->register_block_type();
 		}
 	}
-
-	/**
-	 * Register custom order status for orders created via the API during checkout.
-	 *
-	 * Draft order status is used before payment is attempted, during checkout, when a cart is converted to an order.
-	 *
-	 * @param array $statuses Array of statuses.
-	 * @return array
-	 */
-	public static function register_draft_order_status( array $statuses ) {
```

Comment:
I doubt anyone would be calling these functions directly, but do you think I should leave them as wrappers around the new function calls?

Reply:
I don't see the utility in these for public use. Ideally they would have been private, but filters must be public scoped. Do you think we should add `@internal` or even `@private` docblocks to these methods, even though we're technically prevented from doing this explicitly?